### PR TITLE
Contact Us Section Updated

### PR DESCRIPTION
**Pull request by saggu10417**
I have added some contact ways to asetalias such as twitter, instagram, telegram, facebook and gitlab.
contact us

### CODE REVIEW

**Comment by thisisayush** [comment]
I guess telegram should not be publicly visible to prevent spam, it's a member only thing. What say @vipulgupta2048 
Rest looks good to me

**Comment by AniVerma17** [comment]
> This component could look better if the social links could be rendered in a double column list. Right now, its just breaking the UI.
> I agree with @thisisayush on making the Telegram link public but let's try it and see what happens.

Or instead of full social labels, just display the icons (a bit enlarged with some padding) in a row layout. That'll probably look better and use the width more efficiently.

**Comment by saggu10417** [comment]
@thisisayush @vipulgupta2048 So what should I do as a final step for it?
Should i remove telegram or keep it and what about the icons suggested by @AniVerma17.

**Comment by saggu10417** [comment]
@thisisayush @vipulgupta2048 Any suggestion ?

**Comment by thisisayush** [comment]
I guess we can go as @AniVerma17 suggested and also as @vipulgupta2048 suggested, let's just try it out and see if we get any spam, we'll just remove the link for telegram.

**Comment by thisisayush** [comment]
@saggu10417 Any update?

File: data/events.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [{
+<<<<<<< HEAD
+    "title": "No upcoming events",
```
Comment (vipulgupta2048) [inline data/events.json]: Please refer #132 and work towards making a poster and keeping Thursday meetups as a passive event.

File: communities.html [hunk 2]
```diff
@@ -279,6 +279,26 @@ <h3>Contact us</h3>
                             <i class="fa icon-grey fa-linkedin-square text-blue"></i>
                             <a href="https://www.linkedin.com/company/asetalias/" target="_blank">LinkedIn</a>
                         </li>
+                        <li>
+                            <i class="fa icon-grey fa-telegram text-blue"></i>
+                            <a href="https://www.t.me/asetalias/" target="_blank">Telegram</a>
+                        </li>
+                        <li>
+                            <i class="fa icon-grey fa-gitlab text-grey"></i>
+                            <a href="https://www.gitlab.com/asetalias/" target="_blank">GitLab</a>
```
Comment (vipulgupta2048) [inline communities.html]: Not active there at all.

File: data/events.json [hunk 3]
```diff
@@ -9,6 +21,7 @@
     "location": "E3-104",
     "eventOD": 1,
     "description": "An introduction to the world of programming, CS careers and more",
+>>>>>>> f6d46f649772e5f0acfe9dd1b67bb86cd5e68266
```
Comment (vipulgupta2048) [inline data/events.json]: Remove

File: data/events.json (hunk 1 above)
Comment (saggu10417) [reply data/events.json]: Which type of poster it should be. I mean what should it consist. Which type of details we can keep in it.

File: data/events.json (hunk 3 above)
Comment (saggu10417) [reply data/events.json]: Done

File: communities.html (hunk 2 above)
Comment (saggu10417) [reply communities.html]: Done

File: data/events.json (hunk 1 above)
Comment (vipulgupta2048) [reply data/events.json]: Just a normal poster, regarding Thursday meetups. 
Venue - Ask on the group 
Time - Ask on the group
OD - No
Registration links - N/A
Open For All

Description - An open for all weekly meetup to discuss, ideas, projects and future events with members.

File: index.html [hunk 4]
```diff
@@ -235,50 +235,50 @@ <h1 class="siteAboutContainer animated section-header">
...
+                                <i class="fa fa-thumbs-down icon text-danger"></i>&nbsp;
+                                <span><strong>No OD</strong></span>
                             </div>
                         </div>
                     </div>
                     <div class="row eventCTA">
                         <div class="col-xs-12 text-center">
-                            <a href="https://asetalias.in" target="_blank">
-                                <button class="btn btn-danger btn-lg btn-flat">Open for all freshers, no registration required</button>
+                            <a href="https://docs.google.com/forms/d/e/1FAIpQLScpSM7nisQQKlBB5oYdkeo2cMV6I0b__F_KowLgbUJePyyLZg/viewform?usp=sf_link" target="_blank">
```
Comment (vipulgupta2048) [inline index.html]: @TanyaaCJain @thisisayush  I think we deprecated this form in favor of using the logistics repo as event issues and queries.

**Comment by vipulgupta2048** [comment]
Closing in favor of #165

**Pull request closed without merging**